test/bank_machine/content_test.clj

```diff
@@ -1,189 +1,131 @@
-;; Test File Content Manipulation Functions:    ----------------------------------------------
-;;  - Note: These tests are using the live file, /resources/files/machine_file.txt.
-;;          Therefore, changes made to the live file can be tested prior to running it to make
-;;          sure the file structure is still maintained with the changes in place as well.
 (ns bank-machine.content-test
-  (:require [clojure.test :refer :all]
-            [bank-machine.content :refer :all]
-            [clojure.java.io :as io :refer [resource]]
-  )
-)
+  "Test File Content Manipulation Functions:    ----------------------------------------------
+  - Note: These tests are using the live file, /resources/files/machine_file.txt.
+          Therefore, changes made to the live file can be tested prior to running it to make
+          sure the file structure is still maintained with the changes in place as well."
+  (:require [clojure.test :refer [deftest testing is]]
+            [bank-machine.content :as content]
```

Comment:
Pretty common to avoid using `:refer :all`, since it makes the code a little more explicit later about where functions are coming from (i.e., another namespace).